paddle/fluid/framework/details/all_reduce_op_handle.cc

```diff
@@ -52,7 +52,7 @@ void AllReduceOpHandle::RunImpl() {
 // this is a distributed or inter-process call, find a better way.
 #ifdef PADDLE_WITH_CUDA
   if (NoDummyInputSize() == 1 &&
-      local_scopes_[0]->FindLocalVar(NCCL_ID_VARNAME) == nullptr) {
+      local_scopes_[0]->FindVar(NCCL_ID_VARNAME) == nullptr) {
```

Comment:
Need document here why change to findvar

Reply:
done.